### ISSUE TITLE
Evaluate NNTrainer with full network & unit tests

### ISSUE THREAD

**Issue by jijoongmoon** [nnstreamer/nntrainer#350]
We have to evaluate nntrainer with full network. Compare with tensorflow output.
- [ ] Check Layer by Layer
   - [x] implement python code to generate golden data with tensorflow
   - [x] Conv2D
   - [x] Fully Connected Layer
   - [x] Pooing 2D Layer
   - [x] Flatten Layer
   - [x] Optimizer : SGD
   - [x] Optimizer : ADAM
   - [x] Loss Calculation : MSE
   - [x] Loss Calculation : Cross-sigmoid
   - [x] Loss Calculation : Cross-softmax
   - [x] Weight decay
   - [x] Learning rate decay
   - [x] Batch normalization

- [x] Evaluate with Full Network

- [x] MNIST
   - [x] MNIST implementation ( NNtrainer & tensorflow )
   - [x] Comparison
- [x] VGG (#200)
- [ ] Mobilenet v1 (#201)
- [ ] Mobilenet v2 (#201)

**Comment by jijoongmoon** [comment]
Close and make another issues for mobilenet
